### PR TITLE
[quant]ao migration for quantization mappings and fuser method mappings hg mv

### PR DESCRIPTION
**Pull request by HDCharles**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64985
* #64982

moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

### CODE REVIEW

File: test/quantization/ao_migration/test_fuser_method_mappings.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from torch.testing._internal.common_utils import TestCase
+
+
+class TestFuserMethodMappingsPyImport(TestCase):
```
Comment (supriyar) [inline test/quantization/ao_migration/test_fuser_method_mappings.py:4]: I think so far all the tests have been put into the same file https://github.com/pytorch/pytorch/blob/master/test/quantization/ao_migration/test_quantize.py. We can probably rename the file name so it is more generic.

File: torch/quantization/fuser_method_mappings.py [hunk 2]
```diff
@@ -1,133 +1,15 @@
...
-                                     additional_fuser_method_mapping)
-    fuser_method = all_mappings.get(op_list, None)
-    assert fuser_method is not None, "did not find fuser method for: {} ".format(op_list)
-    return fuser_method
+# flake8: noqa: F401
+r"""
+This file is in the process of migration to `torch/ao/quantization`, and
+is kept here for compatibility while the migration process is ongoing.
+If you are adding a new entry/functionality, please, add it to the
+`torch/ao/quantization/MY_FILE.py`, while adding an import statement
```
Comment (supriyar) [inline torch/quantization/fuser_method_mappings.py:6]: replace MY_FILE with fuser_method_mapping?

File: test/quantization/ao_migration/test_fuser_method_mappings.py (hunk 1 above)
Comment (HDCharles) [reply test/quantization/ao_migration/test_fuser_method_mappings.py]: ya thats been updated

File: torch/quantization/fuser_method_mappings.py (hunk 2 above)
Comment (HDCharles) [reply torch/quantization/fuser_method_mappings.py]: done

**Pull request closed without merging**